src/datasets/features/audio.py

```diff
@@ -174,11 +175,12 @@ def decode_example(
             source_url = path.split("::")[-1]
             try:
                 repo_id = string_to_dict(source_url, config.HUB_DATASETS_URL)["repo_id"]
-                token = token_per_repo_id[repo_id]
+                token_per_repo_id[repo_id]
             except (ValueError, KeyError):
-                token = None
+                pass
```

Comment:
you should revert this no ? we need to get the token from token_per_repo_id

Reply:
it's ok, no need to revert, I just need the change the settings of DownloadConfig, and I will adjust it.